bindings/nodejs/types/account.ts

```diff
@@ -12,30 +13,39 @@ export interface AccountBalance {
     total: string;
     available: string;
     requiredStorageDeposit: string;
-    nativeTokens: nativeTokensMap;
+    nativeTokens: INativeToken[];
     nfts: string[];
     aliases: string[];
     foundries: string[];
-    potentiallyLockedOutputs: potentiallyLockedOutputsMap;
+    /**
+     * Outputs with multiple unlock conditions and if they can currently be spent or not. If there is a
+     * TimelockUnlockCondition or ExpirationUnlockCondition this can change at any time
+     */
+    potentiallyLockedOutputs: Record<string, boolean>;
 }
 
-export type nativeTokensMap = {
-    [tokenId: string]: number;
```

Comment:
I had some time now to read up on this. `Record` and index signatures are fully equivalent. `Record` being `Record<string, number> = { [P in string]: number }`.

However with Record, we now lose the information about the actual meaning of the keys. An example: Devs would not know that inside Record<string, number>, the string represents the `tokenId`. So after coming to this conclusion, I think it's better to undo the changes. What are your thoughts on this?

Reply:
Maybe this should not be done for nativeTokens, since the list approach makes sense (and there is no loss of information). But it might be better to revert it for potentiallyLockedOuputs. If we do that, we should rename the type to `PotentiallyLockedOuputs`, instead of `potentiallyLockedOutputsMap`, since it's not a real map.

Reply:
I agree, I think the named key with the indexed object notation is nice. The name of the key also shows up in the compiled types, unlike a type aliased primitive.